main.go

```diff
@@ -556,10 +556,10 @@ func main() {
 	go func() {
 		defer wg.Done()
 
-		// Do not report this instance as running for the first 10 minutes.
+		// Do not report this instance as running for the first 24 hours.
 		// Among other things, that solves reporting during PMM Server building when we start pmm-managed.
 		// TODO https://jira.percona.com/browse/PMM-4429
```

Comment:
That comment is no longer applies to that huge delay.

It may be worth to move that delay inside the service itself.